Multisig/UI/Safe Management/AddSafeIntroView.swift

```diff
@@ -14,29 +14,35 @@ struct AddSafeIntroView: View {
 
     var body: some View {
         FullSize {
-            Text("Get started by loading your\nSafe Multisig")
-                .padding()
-                .font(.gnoTitle3)
-                .multilineTextAlignment(.center)
-                .foregroundColor(.gnoDarkBlue)
+            VStack(spacing: 21) {
+                Text("Get started by loading your\nSafe Multisig")
+                    .font(.gnoTitle3)
+                    .multilineTextAlignment(.center)
+                    .foregroundColor(.gnoDarkBlue)
 
 
-            Button("Load Safe Multisig") {
-                self.addSafeStarted = true
-            }
-            .buttonStyle(GNOFilledButtonStyle())
-            .sheet(isPresented: self.$addSafeStarted) {
-                EnterSafeAddressView()
-            }
+                Button("Load Safe Multisig") {
+                    self.addSafeStarted.toggle()
+                }
+                .buttonStyle(GNOFilledButtonStyle())
+                .sheet(isPresented: self.$addSafeStarted) {
+                    NavigationView {
+                        SafeAddressForm(form: SafeAddressFormModel())
```

Comment:
why do you need to inject it explicitly?

Reply:
there is no reason why this is done so